app/src/main/kotlin/at/bitfire/davdroid/resource/LocalCalendarStore.kt

```diff
@@ -97,6 +101,14 @@ class LocalCalendarStore @Inject constructor(
         return values
     }
 
+    override fun updateAccount(oldAccount: Account, newAccount: Account) {
+        val values = contentValuesOf(Tasks.ACCOUNT_NAME to newAccount.name)
```

Comment:
`Tasks.ACCOUT_NAME` has the same value and will probably never be changed, but we should reference the correct constant from `CalendarContract` nevertheless.